_extras/guide.md

```diff
@@ -40,12 +40,19 @@ installation. Otherwise, it will tell you that the system is good to go and read
 	`tuple`
 
 ### Dictionaries Challenges
-* Can you do reassignment in a dictionary? Give it a try.
+
+* Changing dictionaries: 2. Reassign the second value.
```

Comment:
1. What does `2.` mean?
2. `Second value` does not sound right as dictionary is not a sequence of values but of key-value pairs.

Reply:
I agree with this "Reassign the value of the second key" might be what we're looking for here.

Reply:
Thank you, April. However, we can not even use "the second key" as dictionaries are not ordered. We have to be very specific with dictionaries and say `value that corresponds to key of <blah>` (blah = 2 in this case).

Reply:
Good catch - easy to forget. I like

"Reassign the value that corresponds to the key "2"."

Reply:
Please don't hate my but... no double quotes around 2 because `2` and `"2"` are two different and legitimate keys.

Reply:
Resolved by #336

Reply:
My mistake, this aspect is not resolved by #336. Could we change this  to 

" Reassign the value that corresponds to the key `2`."

Reply:
Thank you @wrightaprilm and @maxim-belkin. Should be correct now.